cumulusci/tasks/github/release_report.py

```diff
@@ -0,0 +1,83 @@
+from datetime import datetime
+import json
+import pytz
+import re
+import time
+
+from cumulusci.core.utils import process_bool_arg
+from cumulusci.tasks.github.base import BaseGithubTask
+
+
+class ReleaseReport(BaseGithubTask):
+    task_options = {
+        'date_start': {
+            'description': 'Filter out releases created before this date (YYYY-MM-DD)',
+        },
+        'date_end': {
+            'description': 'Filter out releases created after this date (YYYY-MM-DD)',
+        },
+        'include_beta': {
+            'description': 'Include beta releases in report [default=False]',
+        },
+        'print': {
+            'description': 'Print info to screen as JSON [default=False]',
+        },
+    }
+
+    def _init_options(self, kwargs):
+        super(ReleaseReport, self)._init_options(kwargs)
+        self.options['date_start'] = self._parse_datetime(
+            self.options['date_start'],
+        ) if 'date_start' in self.options else None
+        self.options['date_end'] = self._parse_datetime(
+            self.options['date_end'],
+        ) if 'date_end' in self.options else None
+        self.options['include_beta'] = process_bool_arg(
+            self.options.get('include_beta', False))
+        self.options['print'] = process_bool_arg(
+            self.options.get('print', False))
+
+    @staticmethod
+    def _parse_datetime(dt_str):
```

Comment:
please throw a `TaskOptionsError` if invalid format (or something else under the cci base exception class)

Reply:
also consider putting this in utils.py with `process_bool_arg`